grafana/resource_role_assignment.go

```diff
@@ -0,0 +1,152 @@
+package grafana
+
+import (
+	"context"
+	"fmt"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceRoleAssignment() *schema.Resource {
+	return &schema.Resource{
+		Description: `
+**Note:** This resource is available only with Grafana Enterprise 9.2+.
+* [Official documentation](https://grafana.com/docs/grafana/latest/enterprise/access-control/)
```

Comment:
```suggestion
* [Official documentation](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/)
```
The location has changed (we can update the rest of the resources in a separate PR).